### ISSUE TITLE
Feature request: Show newer version available at start

### ISSUE THREAD

**Issue by qdm12** [qdm12/gluetun#135]
1. What's the feature?

Check Github for newer releases/commits and log a message if a newer version is available.

2. Why do you need this feature?

For the user to know if there is an update without having to go to github or docker hub.

3. Extra information?

**Comment by Pimmelton** [comment]
Hi!  Great project.  I spotted this issue while looking around to learn about the many recent changes and thought I would add a few comments.

a)  Please make this configurable in the settings and off by default.  Given the security-related nature of this application, I'd prefer any network traffic to be limited to that generated by user activities and not have unrelated stuff going on in the background.

b)  Probably not worth stating because it's obvious, but do the version check through the VPN after the secure connection has been established, not before during container startup.

c)  Consider just not doing this since it would be easy to accomplish the same thing by including a simple shell command or URL in the project README that people can use when they need to know the latest available version.  Finding the right log message is arguably more work than just doing a quick offline check.  Anyone managing their own containers is likely to know what version they are running anyway or have procedures for updating their infrastructure that already includes a version check.  IMHO.

Thanks for listening!

**Comment by qdm12** [comment]
These are good points. Issue closed :smile:

You also made me think that there are also other DNS over TLS files (block lists, root keys etc.) downloaded at start from Github before tunneling to setup the dns over tls connection (see [here](https://github.com/qdm12/private-internet-access-docker/blob/master/doc/faq.md#what-files-does-it-download-at-start-before-tunneling)). I'll create another issue in order to have these files in the Docker image and update those+restart Unbound only after the VPN has connected. That way PIA subdomains can still be resolved over TLS but you don't need to access github.com before tunneling.

And for this issue, I agree, I'll make it optional although it would just send a single request to github over the tunnel so not so much background traffic. It would be nice combined with #131 although it can also be annoying if you restart the container (without persistence) and it tells you there is a new version all the time :wink:

**Comment by qdm12** [comment]
I added a call to the Github API once the tunnel is up, and it logs a single line showing you if you are running the latest version, if there is a newer release etc. I left it enabled by default as github is still reached to obtain DNS block lists so it doesn't make much sense to disable it. However, you can disable it with `-e VERSION_INFORMATION=off` if you wish to.
